### ISSUE TITLE
SyntaxError: Arg string terminates parameters early

### ISSUE THREAD

**Issue by yoursunny** [kenpb/parcel-plugin-ifdef#2]
This plugin does not work on Windows because some environment variable name contains `(`.

Windows 10 version 2004 has these environment variables by default:
```shell
$ env | grep -E '\(.*='
CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files
ProgramFiles(x86)=C:\Program Files (x86)
```

**package.json**
```json
{
  "private": true,
  "scripts": {
    "start": "parcel main.js --no-autoinstall"
  },
  "devDependencies": {
    "@kenpb/parcel-plugin-ifdef": "^1.1.0",
    "parcel-bundler": "^1.12.4"
  }
}
```

**main.js**
```js
/// #if false
console.log("BAD");
/// #else
console.log("OK");
/// #endif
```

Parcel compilation fails with error:

```
Server running at http://localhost:1234 
×  C:\Users\sunny\Documents\code\parcelbug\main.js: error evaluation #if condition(false): SyntaxError: Arg string terminates parameters early
```

**Comment by yoursunny** [comment]
The error is thrown here:
https://github.com/nippur72/ifdef-loader/blob/5018e6437fb6d8d37568d3003e2c2e6b4dd8ad49/preprocessor.ts#L209

`ifdef-loader` only supports options passed in `webpack.config.js`, and does not inherit `process.env`:
https://github.com/nippur72/ifdef-loader/issues/38
It is not designed to handle a property with `(` in its name.

However, `@kenpb/parcel-plugin-ifdef` passes `process.env`:
https://github.com/kenpb/parcel-plugin-ifdef/blob/6b3809e45857c850a1361b2c3623a117483de951/js-asset.js#L7

I think `@kenpb/parcel-plugin-ifdef` should filter out such environment variables.

**Comment by kenpb** [comment]
Hello @yoursunny, thanks for reporting and digging up the problem, I'll publish the workaround you mentioned later this day.

Cheers.

**Comment by kenpb** [comment]
Hey @yoursunny, just published `1.1.1` give it a test run and let me know how it goes.

**Comment by yoursunny** [comment]
Version `1.1.1` made things worse.
https://travis-ci.com/github/yoursunny/NDNts-NFD-status-page/builds/170423309#L250-L253

* `util.js` is not published: test locally with `npm pack .`, and you'll catch such mistakes.
* `util.js` is ESM, but `ts-asset.js` is `require`-ing it as CommonJS.
* Function names do not match.
* `js-asset.js` does not import `util.js`.

**Comment by kenpb** [comment]
Oops, I even forgot the way env vars are stored, very sorry @yoursunny, thanks for feedback and the PR, published again @`1.1.2`.
